Lombiq.NodeJs.Extensions/Lombiq.NodeJs.Extensions.csproj

```diff
@@ -68,13 +68,16 @@
   <Target Name="SetUpGlobalESLintConfiguration" BeforeTargets="BeforeResolveReferences">
     <PropertyGroup>
       <NodeJsExtensionsGlobalESLintConfigurationDirectory
-        Condition="'$(NodeJsExtensionsGlobalESLintConfigurationDirectory)' != ''
+        Condition="'$(NodeJsExtensionsGlobalESLintConfigurationDirectory)' == ''
```

Comment:
OrchardCore.Commerce\src\OrchardCore.Commerce.Web\.eslintrc.json is created for me upon rebuild.

Reply:
It's not created for me. OrchardCore.Commerce\src\Modules\OrchardCore.Commerce.eslintrc.json isn't created either.

Reply:
With the latest OCC (37d8aa2798dc8741dd80ed19993f4cec218f4c86) it's good for me too.